extension-style/src/main/java/com/mapbox/maps/extension/style/StyleExtensionImpl.kt

```diff
@@ -133,6 +139,7 @@ class StyleExtensionImpl private constructor(
      *
      * @return [Pair]<[Layer], [LayerPosition]>
      */
+    @JvmOverloads
```

Comment:
can we set the JvmName something like `addLayerAtPosition`?

Reply:
Yes, it's a better name